web/packages/teleport/src/Discover/Shared/ConfigureDiscoveryService/ConfigureDiscoveryService.tsx

```diff
@@ -0,0 +1,187 @@
+      const discoveryConfig = await createDiscoveryConfig(clusterId, {
+        name: crypto.randomUUID(),
+        discoveryGroup: discoveryGroupName,
+        aws: [
+          {
+            types: ['rds'],
+            regions: [agentMeta.awsRegion],
+            tags: { 'vpc-id': [agentMeta.vpcId] },
+            integration: agentMeta.awsIntegration.name,
+          },
+        ],
+      });
+      setCreateDiscoveryConfigAttempt({ status: 'success' });
+      updateAgentMeta({
+        ...agentMeta,
+        autoDiscovery: {
+          config: discoveryConfig,
+        },
+      });
+    } catch (err) {
+      const message = getErrMessage(err);
+      setCreateDiscoveryConfigAttempt({
+        status: 'failed',
+        statusText: `failed to create discovery config: ${message}`,
+      });
+      emitErrorEvent(`failed to create discovery config: ${message}`);
+      return;
+    }
+  }
+
+  return (
+    <Box maxWidth="1000px">
+      <Header>Configure Teleport Discovery Service</Header>
+      <EnrollInfo kind={resourceSpec.kind} />
+      <ConfigureDiscoveryServiceDirections
+        showSubHeader={false}
+        clusterPublicUrl={storeUser.state.cluster.publicURL}
+        discoveryGroupName={discoveryGroupName}
+        setDiscoveryGroupName={setDiscoveryGroupName}
+      />
+      <ActionButtons onProceed={handleNextStep} onPrev={prevStep} />
+      {createDiscoveryConfigAttempt.status !== '' && (
+        <CreatedDiscoveryConfigDialog
+          attempt={createDiscoveryConfigAttempt}
+          next={nextStep}
+          close={() => setCreateDiscoveryConfigAttempt({ status: '' })}
+          retry={handleNextStep}
+          region={agentMeta.awsRegion}
+          notifyAboutDelay={false}
+        />
+      )}
+    </Box>
+  );
+}
+
+function EnrollInfo({ kind }: { kind: ResourceKind }) {
+  if (kind === ResourceKind.Database) {
+    return (
+      <Text mb={4}>
+        The Teleport Discovery Service can connect to Amazon RDS and
+        automatically discover and enroll RDS instances and clusters.
+      </Text>
+    );
+  }
+
+  if (kind === ResourceKind.Server) {
+    return (
+      <>
+        <Text mb={4}>
+          The Teleport Discovery Service can connect to Amazon EC2 and
+          automatically discover and enroll EC2 instances.
+        </Text>
+        <OutlineInfo mt={3} linkColor="buttons.link.default">
+          <Box>
+            <InfoIcon />
+          </Box>
+          <Box>
+            Auto discovery will enroll all EC2 instances found in a region. If
+            you want to enroll a <Mark>single</Mark> EC2 instance instead,
+            consider following{' '}
+            <InternalLink
+              to={{
+                pathname: cfg.routes.discover,
+                state: { searchKeywords: 'linux' },
+              }}
+            >
+              the Teleport service installation
+            </InternalLink>{' '}
+            flow.
```

Comment:
Oh sorry, I've just noticed there's a "flow" after the link. I think it could be included in the anchor as well? 😅 Since it's a single thing we link to, the Teleport service installation flow.